randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -140,12 +142,19 @@ Extra - asset_id: 3
   * Normal Door to West Ocean/Door to Moat
   > Door to Crateria Kihunter Room
       Any of the following:
-          # Via top path - TODO: check for shinespark without entrance rando?
-          Grapple Beam or Space Jump
+          Any of the following:
+              # Via top path
+              Grapple Beam or Space Jump or Movement (Intermediate)
+              # Start in West Ocean and Jump Over with momentum
+              Speed Booster
           Any of the following:
               # Via bottom path
               Can Lay Bombs
-              Gravity Suit and Screw Attack
+              All of the following:
+                  Morph Ball
+                  Any of the following:
+                      Enabled Moat Tweaks
+                      Gravity Suit and Screw Attack
```

Comment:
since screw does work underwater if you spinjumped from outside the water, i wonder whether that should be locked behind some knowledge trick. thoughts?

Reply:
I'm not sure I follow. I have gravity suit added here so the player can screw attack under water.

Reply:
The gravity is for using screw *under water*.
AFAIK, if you jump from *out of water* into the water with screw attack, it should still give you the screwy effect around samus.

Reply:
You're right on the screw attack tech, but for some reason screw attack doesn't break this block. I've tried with and without gravity. Removed it as an option.